README.md

```diff
@@ -1,51 +1,110 @@
 
-## Installation
+```shell
+pip install qualtran
+```
+
+You can also install the latest version of the main branch on GitHub:
 
-Qualtran is being actively developed. We recommend installing from source:
+```shell
+pip install git+https://github.com/quantumlib/Qualtran
+```
 
-For a local editable copy:
+## Usage
 
-    git clone https://github.com/quantumlib/Qualtran.git
-    cd Qualtran/
-    pip install -e .
+> [!WARNING]
+> Qualtran is an experimental preview release. We provide no backwards compatibility guarantees.
+> Some algorithms or library functionality may be incomplete or contain inaccuracies. Open issues or
+> contact the authors with bug reports or feedback.
 
-You can also install the latest tagged release using `pip`:
+### Python interpreter and programs
+
+You should be able to import the `qualtran` package into your interactive Python environment as
+as well as your programs:
+
+```shell
+import qualtran
+```
 
-    pip install qualtran
+If this is successful, you can move on to learning how to
+[write bloqs](https://qualtran.readthedocs.io/en/latest/_infra/Bloqs-Tutorial.html) or investigate
+the [bloqs library](https://qualtran.readthedocs.io/en/latest/bloqs/index.html#bloqs-library).
+
+### Physical Resource Estimation GUI
+
+Qualtran provides a GUI for estimating the physical resources (qubits, magic states, runtime, etc.)
+needed to run a quantum algorithm. The GUI can be run locally by running:
+
+```shell
+cd $QUALTRAN_HOME
+python -m qualtran.surface_code.ui
+```
+
+## Documentation
 
-You can also install the latest state of the main branch:
+Documentation is available at https://qualtran.readthedocs.io/.
 
-    pip install git+https://github.com/quantumlib/Qualtran
+## News
 
-## Physical Resource Estimation GUI
+Stay on top of Qualtran developments using the approach that best suits your needs:
 
-Qualtran provides a GUI for estimating the physical resources (qubits, magic states, runtime, ..etc) needed to run a quantum algorithm. The GUI can be run locally by running:
+*   For news and updates announcements: sign up to the low-volume mailing list
+    [`qualtran-announce`].
+*   For releases only:
+    *   *Via GitHub notifications*: configure [repository notifications] for Qualtran.
+    *   *Via RSS from GitHub*: subscribe to the GitHub [Qualtran releases feed].
+    *   *Via RSS from PyPI*: subscribe to the [PyPI releases feed] for Qualtran.
 
-    cd $QUALTRAN_HOME
-    python -m qualtran.surface_code.ui
+[`qualtran-announce`]: https://groups.google.com/g/qualtran-announce
+[repository notifications]: https://docs.github.com/github/managing-subscriptions-and-notifications-on-github/configuring-notifications
+[Qualtran releases feed]: https://github.com/quantumlib/Qualtran/releases.atom
+[PyPI releases feed]: https://pypi.org/rss/project/qualtran/releases.xml
 
 ## Citation
 
 When publishing articles or otherwise writing about Qualtran, please cite the
 following:
 
-```latex
+```bibtex
```

Comment:
RE the "Contact" section: recommend mentioning also the option of joining the bi-weekly call

Reply:
Excellent idea. Will do.

Reply:
sgtm